### PR TITLE
Model Data Refactor

### PR DESCRIPTION
**Pull request by trevorlbrooks**
Refactored database calls in the model.

### CODE REVIEW

File: backend/model/dbStartup.php.smp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+  $dsn = 'mysql:host=localhost;dbname=';
+  $username ='';
+  $password = '';
```
Comment (HumbleHominid) [inline backend/model/dbStartup.php.smp]: Just don't put credentials on here on accident.

File: backend/model/events.php [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
-include('dbStartup.php');
 class Events{
 
    private $login = NULL;
    private $members = NULL;
    private $files = NULL;
+  private $conn = NULL;
 
    function Events($login, $members){
+    $this->conn = new DbConn();
```
Comment (HumbleHominid) [inline backend/model/events.php]: Weird white space things you might want to fix

File: backend/model/fees.php [hunk 3]
```diff
@@ -125,12 +108,9 @@ function memberFees($user_id){
...
 
    function getAllFees(){
-      include('dbStartup.php');
+     
       $query = 'SELECT DISTINCT debtor_id FROM Debtor_Fees;';
-      $statement = $db->prepare($query);
-      $statement->execute();
-      $results = $statement->fetchAll(PDO::FETCH_ASSOC);
-      $statement->closeCursor();
+        $results = $this->conn->select($query);
```
Comment (HumbleHominid) [inline backend/model/fees.php]: another weird thing.

File: backend/model/files.php [hunk 4]
```diff
@@ -1,26 +1,23 @@
...
    private $members = NULL;
+    private $conn = NULL;
 
    function Files($login, $members){
-      $this->login = $login;
-      $this->members = $members;
 
+     $this->conn = new DbConn();
+     $this->login = $login;
+      $this->members = $members;
```
Comment (HumbleHominid) [inline backend/model/files.php]: here too

File: backend/model/members.php [hunk 5]
```diff
@@ -1,24 +1,20 @@
 <?php
-include('dbStartup.php');
 class Members{
+    private $conn = NULL;
+  function Members(){
```
Comment (HumbleHominid) [inline backend/model/members.php]: same

File: backend/model/officers.php [hunk 6]
```diff
@@ -1,47 +1,42 @@
...
 class Officers{
-  
+
   private $login = NULL;
   private $members = NULL;
+  private $conn = NULL;
 
   function Officers($login, $members){
+
+   $this->conn = new DbConn;
```
Comment (HumbleHominid) [inline backend/model/officers.php]: same same

File: backend/model/dbStartup.php.smp (hunk 1 above)
Comment (trevorlbrooks) [inline backend/model/dbStartup.php.smp]: Yeah. That would be bad. Wanted to include a sample config file though for open source reasons.

Comment (HumbleHominid) [inline backend/model/dbStartup.php.smp]: 👍

**Pull request merged**